### PR TITLE
Cricut .SVG Files for Voron Design Logos

### PR DESCRIPTION
**Pull request by GadgetAngel**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

printer_mods/

#### How was it tested? 

Using my Cricut Maker 3 machine

#### Any background context you want to provide?

I wanted to add an extra touch to my Voron 2.4 Build.  I wanted to place my own LOGO for the printer on the Back panel of the Voron 2.4 but I had a hard time finding the vector files to do this.  I looked around for vector files of the Voron Design Parts Logo and found that the vector (.svg) files were not easy to find.  

A lot more people have access to Cricut cutting machines and can create their own logo or re-create the Voron Design Parts Logo and place the Vinyl on the appropriate panel for their Voron Build.

But finding a vector file that has minimal anchor points (or the number of cuts to perform the Part Design Logo is small enough so the Cricut Design Space software can handle the project) is not easy.  I have spent over one month creating these .svg (or vector) files.

This is my fourth iteration of trying to create vector files for the Voron Design Logos. I first tried to use Adobe Photoshop to create the vector files. But the vector files had too many anchor points.  I tried to use the Inkscape Program to create the vector paths but Inkscape created too many anchor points and the shapes were not detailed enough for my taste.  I tried using the trace function in Adobe Illustrator but again it produced too many anchor points.  So I finally decide that the best way was to hand trace the vectors out.  The pictures you see below are the .png or .jpg files I produced using Adobe Illustrator after the vector lines are filled in with a solid color.  

To better explain what I mean by too many anchor points, the Hexagon shape should only need 12 anchor points to define the shape. Two anchor points for each corner as the blue anchor points show in the picture below: 
Minimal_Vector_points_for_Hexagon_Shape

All other methods I tried produced a lot more than just 12 anchor points to create the Hexagon shape.  But if you hand traced the Hexagon with the "Pen" tool in Adobe Illustrator you can create this shape with only 12 anchor points.  So hand tracing (even tho it is tedious) produces the least amount of anchor points (or cuts points) for the vector shape.

I literally went in and traced each part by hand in Adobe Illustrator to minimize the number of anchor points.

To obtain the Switch Wire Parts Logo, I imported the .stl files into my slicer and laid them out in the Hexagon Design then I sliced the parts to produce only 4mm of height and took a high quality picture of the slicer output.  I then loaded the .jpg file into Adobe Illustrator and again hand traced each part to obtain a brand new SwitchWire Design Parts Logo.

I am just trying to provide other users in the Voron Community an opportunity to build on the vector files I have created.
#### Screenshots (if appropriate)

Here are pictures of the Voron Designs I have created or re-created from others work:

For the Voron Plain Logo:

Voron_Design_Plain_Logo

---

For Voron 2.4 Build:
Voron2 4_1Color

Voron2 4_1Color_WithOut_Logo_Outline_With_Silhouette

Voron2 4_1Color_WithOut_Logo_Outline_WithOut_Silhouette

Voron2 4_3Color

---

For Voron 0.0 or Voron 0.1 Build:

Combined_Red_Path_300ppi

---

For Voron SwitchWire Build:

Cricut_VoronSW_Logo_RearPanel_1Layer

---

For my Voron 2.4 Build:

QueenWithPlainVoronLogo2

QueenWithPlainVoronLogo

QueenWithComplexVoronLogo

---

### CODE REVIEW

**Review by spawn9859 — APPROVED** [review]
I did this to my Switchwire and I love the idea of the parts making the logo. I used holographic vinyl and I'll attach a [picture](https://imgur.com/nMFg4C2) of the finished product, I didn't use the SW parts logo, but the first one.

**Comment by GadgetAngel** [comment]
> I did this to my Switchwire and I love the idea of the parts making the logo. I used holographic vinyl and I'll attach a [picture](https://imgur.com/nMFg4C2) of the finished product, I didn't use the SW parts logo, but the first one.

Yah, I think a lot of users will want to personalize their 3D printers.  In the future I will work on the Trident and Legacy Parts Logo

File: printer_mods/README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ like so:
 |  | [Voron 2.4 cable clamps](./evandepol/Voron2.4-Cable-clamps) | A set of 2020-mounted cable guards to organize and dress the wiring on your build | :x: | :x: | :heavy_check_mark: |:x:|
 | falo | [Magnetic Exhaust Grill Cover](./falo/magnetic_grill_cover) | A magnetic cover to seal the filtered exhaust | :x: | :x: | :heavy_check_mark: | :x: |
 | FreeformFez | [V1.8 Filtered Power Inlet](./FreeformFez/FilteredPower) | V1.6 Filtered Inlet and Rocker Switch for V1.8 | :x: | :heavy_check_mark: | :x: |:x:|
+| GadgetAngel | [Cricut_Voron_Logos](./GadgetAngel/Cricut_Voron_Logos) |For any cutting machines or machines that relies on vector graphics and uses .svg files or vector files. These vectors or .svg files are the Voron Design Logos for the Voron 2.4, Voron SwitchWire and Voron 0.0/0.1. These .svg files use a minimal amount of anchor points to acheive the vector shapes! | :heavy_check_mark: | :x: | :heavy_check_mark: | :heavy_check_mark: | :x: |
```
Comment (KiloQubit) [inline printer_mods/README.md:116]: Please remove underscores inside the square brackets, that's the text shown to the user:

     [Cricut Voron Logos](./GadgetAngel/Cricut_Voron_Logos)

Replace `machines that relies on vector graphics and uses .svg files` with `machines that rely on vector graphics and use .svg files`.

Replace `acheive` with `achieve`

File: printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
+# Cricut_Voron_Logos
+This repository contains vector files I have created for the Voron community and for my own Voron builds.
+
+---
+
+# Puropse of this repository
```
Comment (KiloQubit) [inline printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md:6]: Replace `Puropse` with `Purpose`.

File: printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+The purpose of this repository to share vector files that I have created for my Voron Builds.
```
Comment (KiloQubit) [inline printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md:8]: Add `is` after `repository`.

File: printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+## For "Voron 2.4 Build with 1 Color"
+
+### Link to Cricut Design Space for "Voron 2.4 1_Color_Layer":
+### https://design.cricut.com/landing/project-detail/6137795f68f6f90001d7abfd
+
+### Link to my Repository on Github that contains all files I used to produce the "Voron 2.4 1_Color_Layer" .svg file: https://github.com/GadgetAngel/Cricut_Voron_Logos/tree/main/Voron_2.4_Logo/1_Color_Layer/Current%20Design%20Files
+
+### Picture of "Voron 2.4 Build with 1 Color":
+<img src="https://github.com/GadgetAngel/VoronUsers/blob/Cricut_Voron_Logos_by_GadgetAngel/printer_mods/GadgetAngel/Cricut_Voron_Logos/images/Voron2.4_1Color.jpg?raw=true" />
```
Comment (KiloQubit) [inline printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md:21]: Please change all images to use relative references to this repository.  So this image for example would become

    Voron2.4 1 Color

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please see the readme changes below, biggest things is change all the images to relative references

File: printer_mods/README.md (hunk 1 above)
Comment (GadgetAngel) [reply printer_mods/README.md:116]: done

Comment (KiloQubit) [reply printer_mods/README.md:116]: Did you push your changes?  I don't see any updates on the PR.

File: printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md (hunk 4 above)
Comment (GadgetAngel) [reply printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md]: done for all images

File: printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md (hunk 2 above)
Comment (GadgetAngel) [reply printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md]: done

File: printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md (hunk 3 above)
Comment (GadgetAngel) [reply printer_mods/GadgetAngel/Cricut_Voron_Logos/README.md]: done

**Comment by GadgetAngel** [comment]
All changes requested have been completed

File: printer_mods/README.md (hunk 1 above)
Comment (GadgetAngel) [reply printer_mods/README.md]: Yes, all updates have now been pushed to the branch

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**